### DISCUSSION TITLE
Welcome to TeamsACS Discussions!

### DISCUSSION THREAD

**Discussion by jamiesun** [CA17/TeamsACS#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shalanin** [comment]
Hello!

Is there an API to the TeamsACS server?
Is there a documentation to that API?

**Comment by jamiesun** [comment]
https://github.com/CA17/TeamsACS/wiki
